build.gradle

```
@@ -23,20 +23,20 @@ buildscript {
         targetSdkVersion = 28
 
         // App dependencies
-        accompanistVersion = "0.2.0"
+        accompanistVersion = '0.2.3'
         appCompatVersion = '1.1.0'
-        composeVersion = '1.0.0-alpha01'
+        composeVersion = '1.0.0-alpha04'
         constraintLayoutVersion = '2.0.0-beta3'
         coreTestingVersion = '2.1.0'
         coroutinesVersion = "1.3.9"
         espressoVersion = '3.2.0'
         fragmentVersion = '1.1.0-alpha09'
         glideVersion = '4.10.0'
-        gradleVersion = '4.2.0-alpha08'
+        gradleVersion = '4.2.0-alpha12'
```

My understanding is that we'll need AGP alpha 13 to make things work
```suggestion
        gradleVersion = '4.2.0-alpha13'
```